libs/dao/src/main/java/com/akto/dao/testing/DeleteRunsInfoDao.java

```diff
@@ -0,0 +1,88 @@
+package com.akto.dao.testing;
+
+import java.util.List;
+import java.util.Map;
+
+import org.bson.conversions.Bson;
+import org.bson.types.ObjectId;
+
+import com.akto.dao.AccountsContextDao;
+import com.akto.dao.testing_run_findings.TestingRunIssuesDao;
+import com.akto.dto.test_run_findings.TestingRunIssues;
+import com.akto.dto.testing.DeleteRunsInfo;
+import com.akto.dto.testing.TestingRunResult;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Updates;
+
+public class DeleteRunsInfoDao extends AccountsContextDao<DeleteRunsInfo> {
+
+    public static final DeleteRunsInfoDao instance = new DeleteRunsInfoDao();
+
+    private static Map<String, Boolean> deleteTestRunsCollections(List<ObjectId> testRunIds,
+            List<ObjectId> latestSummaryIds, List<Integer> testConfigIds,
+            Map<String, Boolean> testingCollectionsDeleted) {
+                
+        if (!testingCollectionsDeleted.getOrDefault("testing_run_result", false)) {
```

Comment:
are all these 5 queries dependent on each other? if not then execute them parallely

Reply:
There's a tree in dependency. Our testing has a tree like structure hence not fired parallely